src/components/Card/Description/description.css

```diff
@@ -0,0 +1,30 @@
+@font-face {
+  font-family: 'robotoRegular';
```

Comment:
I feel like font family might need to be defined at a higher level in the project. (specifically for generic text) although it might make sense to explicitly define it for cards so idk.